### ISSUE TITLE
Add FAQ or detail in prerequisites: Does Microsoft Edge with IE mode require Internet Explorer to be installed

### ISSUE THREAD

**Issue by ahsteele** [MicrosoftDocs/EdgeEnterprise#17]
It is unclear and I've had several discussions on whether Internet Explorer is required to be installed to use Edge with IE mode. Please add definitive information to either the FAQ or the prerequisites section.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: aae869a8-2ae4-d28f-6c85-c1190609f87b
* Version Independent ID: e2bcb1d6-27f7-5e30-ae6b-7acb5ee94098
* Content: [Use Microsoft Edge with IE mode](https://docs.microsoft.com/en-us/deployedge/edge-ie-mode)
* Content Source: [edgeenterprise/edge-ie-mode.md](https://github.com/MicrosoftDocs/EdgeEnterprise-pr/blob/live/edgeenterprise/edge-ie-mode.md)
* Product: **microsoft-edge**
* GitHub Login: @dan-wesley
* Microsoft Alias: **kvice**

**Comment by shisub** [comment]
Thanks for reaching out! For the moment, Internet Explorer is required to be installed to use Edge with IE mode. Updated here - https://docs.microsoft.com/en-us/deployedge/edge-ie-mode#error-message-to-open-this-page-in-internet-explorer-mode-reinstall-microsoft-edge-with-administrator-privileges

**Comment by ahsteele** [comment]
@shisub, thank you for answering my question.

Further, I appreciate the work that @dan-wesley did in d8e2e99 to the [error message section](https://docs.microsoft.com/en-us/deployedge/edge-ie-mode#error-message-to-open-this-page-in-internet-explorer-mode-reinstall-microsoft-edge-with-administrator-privileges) which clarified that “Internet Explorer 11 needs to be enabled in Windows Features.”

I’m a little concerned though as nothing was added to the [prerequisite section](https://docs.microsoft.com/en-us/deployedge/edge-ie-mode#prerequisites) or to the FAQ. Having to find this crucial piece of information buried in a troubleshooting section seems counter intuitive. To that end having a definitive answer I went ahead and created pull request #35 with what I think will add clarity.

<sub>Finally, unless I’m missing something the pull request for commit fbac49e that was linked to this issue by @dstrome seems to have been done so in error (unrelated localization stuff).</sub>
